airflow/scripts/git_scripts/aggregate_user_activty.py

```diff
@@ -0,0 +1,94 @@
+
+import requests
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+
+token = 'put_token_here'   # add your token
+
+headers = {'Authorization': f'token {token}', 'Accept': 'application/vnd.github.v3+json'}
+
+# Define the date range
+since = (datetime.now() - timedelta(days=7)).isoformat()
+until = datetime.now().isoformat()
+
+
+def fetch_data(url, params=None):
+    """Fetch data from GitHub API"""
+    response = requests.get(url, headers=headers, params=params)
+    return response.json()
+
+
+def get_commits(owner,repo):
+    """Fetch number of commits"""
+    url = f"https://api.github.com/repos/{owner}/{repo}/commits"
+    params = {'since': since, 'until': until}
+    return fetch_data(url, params)
+
+
+def get_pull_requests(owner,repo):
+    """Fetch the number of PR"""
+    url = f"https://api.github.com/repos/{owner}/{repo}/pulls"
+    params = {'state': 'all', 'sort': 'created', 'direction': 'desc', 'since': since}
+    return fetch_data(url, params)
+
+
+def get_issues(owner,repo):
+    """Fetch the number of issues"""
+    url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+    params = {'since': since}
+    return fetch_data(url, params)
+
+
+def aggregate_user_activity(commits, pull_requests, issues):
+    """Aggregate and count the number of commits, pull requests, and issues created by each user."""
+    user_activity = defaultdict(lambda: {'commits': 0, 'prs': 0, 'issues': 0})
+
+    for commit in commits:
```

Comment:
What will happen if we insert a non-existent user?
would it break this function?